.github/workflows/release-swift.yml

```diff
@@ -84,11 +84,16 @@ jobs:
         id: set-release-name
         run: |
           if [[ ${{ inputs.pre-release }} == true ]]; then
-            echo "release_name=${{ env._PKG_VERSION }}-unstable-${{ needs.validate.outputs.short_sha }}" >> $GITHUB_OUTPUT
+            echo "release_name=${{ steps.version.outputs.version }}-unstable-${{ steps.set-sha.outputs.short_sha }}" >> $GITHUB_OUTPUT
           else
-            echo "release_name=${{ env._PKG_VERSION }}" >> $GITHUB_OUTPUT
+            echo "release_name=${{ steps.version.outputs.version }}" >> $GITHUB_OUTPUT
           fi
 
+      - name: Calculate swift file checksum
+        run: |
+          CHECKSUM=$(swift package compute-checksum BitwardenFFI-${{ steps.version.outputs.version }}-${{ steps.set-sha.outputs.short_sha }}.xcframework)
```

Comment:
Is this the hash of the zip file? I always run the compute-checksum on the already zipped framework.

Reply:
Okay, I thought that needs to be on pre-zipped file. I'll change that.